packs/pfs-season-1-bestiary/greater-metallic-sod-hound.json

```diff
@@ -265,6 +265,9 @@
         }
     ],
     "name": "Greater Metallic Sod Hound",
+    "prototypeToken": {
+        "name": "Metallic Sod Hound"
```

Comment:
What is this fixing?

Reply:
Was just using the same pattern that's used for some elite variants. I'll back it out.